docs/index.md

```diff
@@ -23,18 +22,13 @@ Freqtrade is a cryptocurrency trading bot written in Python.
 
 ## Features
 
- - Based on Python 3.6+: For botting on any operating system — Windows, macOS and Linux.
- - Persistence: Persistence is achieved through sqlite database.
- - Dry-run mode: Run the bot without playing money.
- - Backtesting: Run a simulation of your buy/sell strategy with historical data.
- - Strategy Optimization by machine learning: Use machine learning to optimize your buy/sell strategy parameters with real exchange data.
- - Edge position sizing: Calculate your win rate, risk reward ratio, the best stoploss and adjust your position size before taking a position for each specific market.
- - Whitelist crypto-currencies: Select which crypto-currency you want to trade or use dynamic whitelists based on market (pair) trade volume.
- - Blacklist crypto-currencies: Select which crypto-currency you want to avoid.
- - Manageable via Telegram or REST APi: Manage the bot with Telegram or via the builtin REST API.
- - Display profit/loss in fiat: Display your profit/loss in any of 33 fiat currencies supported.
- - Daily summary of profit/loss: Receive the daily summary of your profit/loss.
- - Performance status report: Receive the performance status of your current trades.
+ 1. Download markets datas : download historical datas of the exchange and the markets your may want to trade with. 
+ 2. Select markets : create your list or use an automatic one based on top traded volume (not available during backtesting). You can blacklist markets you don't want to trade.
+ 3. Backtest : Test your strategy on past datas (based on [ohcl](https://en.wikipedia.org/wiki/Open-high-low-close_chart) candles).
```

Comment:
I think the colon (:) should not have a leading whitespace ... that'll look odd when rendered